<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLineDlg</class>
 <widget class="QDialog" name="QLineDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>直线</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>271</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>线条</string>
     </property>
     <widget class="QComboBox" name="Line_Type">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>26</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
       <property name="icon">
        <iconset resource="../../resourse.qrc">
         <normaloff>:/lineType/images/LineType/0.ico</normaloff>:/lineType/images/LineType/0.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
       <property name="icon">
        <iconset resource="../../resourse.qrc">
         <normaloff>:/lineType/images/LineType/L1.ico</normaloff>:/lineType/images/LineType/L1.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
       <property name="icon">
        <iconset resource="../../resourse.qrc">
         <normaloff>:/lineType/images/LineType/L2.ico</normaloff>:/lineType/images/LineType/L2.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
       <property name="icon">
        <iconset resource="../../resourse.qrc">
         <normaloff>:/lineType/images/LineType/L3.ico</normaloff>:/lineType/images/LineType/L3.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
       <property name="icon">
        <iconset resource="../../resourse.qrc">
         <normaloff>:/lineType/images/LineType/L4.ico</normaloff>:/lineType/images/LineType/L4.ico</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
       <property name="icon">
        <iconset resource="../../resourse.qrc">
         <normaloff>:/lineType/images/LineType/L5.ico</normaloff>:/lineType/images/LineType/L5.ico</iconset>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>26</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>线类型：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>56</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>线宽度：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Line_Width">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>56</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>86</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>线颜色：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Line_Color">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>86</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>透明度：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Alpha">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>末端箭头：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="End_Arrow">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/lineType/images/LineType/0.ico</normaloff>:/lineType/images/LineType/0.ico</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/lineType/images/LineType/1.ico</normaloff>:/lineType/images/LineType/1.ico</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/lineType/images/LineType/2.ico</normaloff>:/lineType/images/LineType/2.ico</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/lineType/images/LineType/3.ico</normaloff>:/lineType/images/LineType/3.ico</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/lineType/images/LineType/4.ico</normaloff>:/lineType/images/LineType/4.ico</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/lineType/images/LineType/5.ico</normaloff>:/lineType/images/LineType/5.ico</iconset>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ID：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinbox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
